data/Crop/cropped.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
1. We shouldn't see a data file being added. It should be shown as rename in the git history. The number of lines changed in git history should be relatively small.
2. Please follow the structure in https://github.com/isl-org/Open3D/tree/yixing/data-dir-mockup/data as we discussed. The local data should be placed in `data/common`.

Reply:
1. Moving it around in VSCode led to delete/add log in git. I've changed it now to rename.
2. Placed the local data in data/common